### DISCUSSION TITLE
Covers in the tree

### DISCUSSION THREAD

**Discussion by Murmel99** [gerbera/gerbera#1823] [Q&A] [answered]
Hi, I am Peter and currently testing and customizing Gerbera (vers. 1.9.0 on Debian 10/Buster). A little bit complicated but very flexible. Everything is fine until now but I have one annoying problem. 

If I use the default layout (“builtin”) the tree looks like this:
![default](https://user-images.githubusercontent.com/89473616/130656787-880bdced-6add-4689-857f-a7f3159f6618.JPG)

But if I switch to “js” to customize gerbera  an album cover is displayed on all nodes, like genre and year:
![JS](https://user-images.githubusercontent.com/89473616/130656820-69760561-8630-4cb1-9510-5dd401e4cc16.JPG)

Is there an option to disable this behavior and enable it only for albums? I could not find an option in the documentation, maybe I just overlooked that. 

Greetings from Germany,
Peter

**Comment by KarlStraussberger** [comment]
Yes, there is an option and yes, it is hidden. It is `parentCount` which is not evaluated in buildin layout (because the metadata is not passed from import to the tree). But it cannnot be disabled for particular items (the folder does not know whats in it). I like the images in the tree and when browsing.

If you want to get rid of the images for **Year** and **Genre** you need to adjust the `res` property in `common.js / addAudio`

https://docs.gerbera.io/en/stable/config-import.html#container

German fellows should use OpenSuSE 🐧

**Reply by KarlStraussberger** [reply]
Here, some more hints that may help you:

You can copy the function to a new file and configure that file with https://docs.gerbera.io/en/stable/config-import.html#custom-script. The functions in the custom script overwrite functions from the common file without changing files shipped with gerbera.

**Comment by Murmel99** [comment, marked as answer]
Hi Karl,

thanks for your fast reply and the hints. I found an easy solution last night based on your tipp to adjust the res property.

It looks now as it should:
![JSneu](https://user-images.githubusercontent.com/89473616/130767288-9159750e-9c40-4631-9267-9328c1f2eb69.JPG)

I only removed the part "res: obj.res" in the "const chain = ..." section for genre, year, artist etc. and not for album.

Could this have some unwanted impact?

Peter

**Reply by KarlStraussberger** [reply]
No. The trick is to transport stuff like album uri from the media file to the container. If nothing is there it does not get data. You may explicitly remove M_ALBUMARTURI from the resources to keep details like artist etc.

**Comment by Murmel99** [comment]
Thanks again, everything is working now and I could build my own structure.

One additional question: For Raspberry Pi only version 1.1.0 is available. Will/can you provide a newer version in the near future?

Peter

**Reply by KarlStraussberger** [reply]
We do already provide several images by our means. To support more distros we need some folks who work out the build environment and set up the proper hosting. So feel free to start with that - now

**Comment by Murmel99** [comment]
I would like to, but I don't have the necessary knowledge. I was an application developer, in the last century. For my first project I had to use punchcards and for the last own project floppy disks. There is a big gap in my knowledge with today's technology. It's just a hobby and I'm already happy when my cloud at home is up and running :-)

Peter
